uninstall.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Undo Step 3: Removing copied config and local folders
+echo 'Removing copied config and local folders...'
+rm -rf "$HOME/.config"
+rm -rf "$HOME/.local"
+
+# Undo Step 2: Uninstall AGS
+echo 'Uninstalling AGS...'
+sudo meson uninstall -C ~/ags/build

```

Comment:
ags isn't here though? 
1. the install script is meant to be run from the repo folder, not home
2. ags is actually something in /usr/bin/ags... but that's just the launcher script. i'm not sure how we can actually get rid of the other libraries

Reply:
Ah well, I've not had much experience with ags
We could just comment that out for now, and enable it after we get a viable solution to remove ags in it's entirety. Until then, I'll create a issue for this.